src/Module/WebFinger.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Core\System;
+use Friendica\Network\Probe;
+
+/**
+ * Web based module to perform webfinger probing
+ */
+class WebFinger extends BaseModule
+{
+	public static function init()
+	{
+		if (!local_user()) {
+			System::httpExit(
+				403,
+				[
+					'title'       => L10n::t("Public access denied."),
```

Comment:
Code standards: Please normalize the quote style in this file.

Reply:
The `L10n::t()` string should get normalized too? Because I recognized troubles sometimes because of encoded formatings inside `t`, where we need double quotes.

Reply:
There used to be an issue with translated strings containing parentheses which prompted the use of `\x28` and `\x29` that can only be used with double-quotes. But this predicament is over and parentheses can be normally used with single-quotes.

Reply:
I see